orion/sites/admin/src/announce/announce.styl

```diff
@@ -1,13 +1,48 @@
 @import "../static.styl"
 
+
 #view-announce
   width: 100%
   height: 400px
   padding: 0 spacing2x
   box-sizing: border-box
-  color: colorBlackDark
+  color: colorGrayDark
+  font-family: fontFamilyRoboto /*not working*/
 
   & h1
     margin: spacing4x 0
     line-height: lineHeight3
     font-size: fontSize3
+  & ul
+    padding: 0
+
+  .announce-lists
```

Comment:
If there is an indent here, please add a & before it.
```
#view-announce
    & .announce-lists
```

Reply:
I'm not sure I understand what you mean. Do you mean to indent the ul or and & and indent the .announce-lists?